src/injected/analyzer-controller.ts

```diff
@@ -65,8 +67,12 @@ export class AnalyzerController {
     };
 
     protected startScan = (id: string): void => {
+        this.shadowInitializer.removeExistingShadowHost();
```

Comment:
It looks like this is also called as part of `shadowInitializer.initialize()`, is there a reason it needs to be called before `analyzer.analyze()` (or for `initialize()` to be called after)?

Reply:
Yeah, analyze actually runs axe-core, so this way we're clearing the visualizations beforehand so that they don't interfere with our axe results, then after we've gotten the results we can safely re-init visualizations